File: src/main/java/org/kane/domain/DTO/entityDTO/diary/meal/MealEditDTO.java
package org.kane.domain.DTO.entityDTO.diary.meal;

import lombok.Data;
import org.kane.database.enum_types.MealType;
import org.kane.domain.DTO.entityDTO.diary.meal_item.MealItemEditDTO;

import java.time.LocalDate;
import java.time.LocalTime;
import java.util.List;

@Data
public class MealEditDTO {
    private Long mealID;
    private MealType mealType;
    private LocalTime time;
    private List<MealItemEditDTO> mealItemEditDTOList;
}

File: src/app/DTO/entity_dto/diary/meal/meal-edit.dto.ts
import {MealType} from '../../../types';
import MealItemEditDTO from '../meal_item/meal-item-edit.dto';

export default interface MealEditDTO {
  mealID: number;
  mealType: MealType;
  time: string;
  mealItemEditDTOList: MealItemEditDTO[];
}
